builtin/settingtypes.txt

```diff
@@ -1713,6 +1713,13 @@ world_aligned_mode (World-aligned textures mode) enum enable disable,enable,forc
 #    Warning: This option is EXPERIMENTAL!
 autoscale_mode (Autoscaling mode) enum disable disable,enable,force
 
+#    Side length of a cube of map blocks that the client will consider together
+#    when generating meshes.
+#    Larger values increase the utilization of the GPU by reducing the number of
+#    draw calls, benefiting especially high-end GPUs.
+#    Systems with a low-end GPU (or no GPU) would benefit from smaller values.
+client_mesh_chunk (Client Mesh Chunksize) int 4 1 10
```

Comment:
I suggest 1 as the default for backwards compatibility. We don't change the default view range, and chunk size impacts performance for certain hardware.

Reply:
OK... I'll also raise the top to 16 (for real high-end cards)